cmd/kyma/test/logs/cmd.go

```diff
@@ -0,0 +1,143 @@
+package logs
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/kyma-project/cli/internal/logs"
+	"strings"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/cmd/kyma/test"
+	"github.com/kyma-project/cli/internal/cli"
+	"github.com/kyma-project/cli/internal/kube"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	defaultIgnoredContainers = []string{"istio-init", "istio-proxy", "manager"}
+	defaultLogsInStatus      = string(oct.TestFailed)
+)
+
+type command struct {
+	opts *options
+	cli.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: cli.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:   "logs <test-suite-1> <test-suite-2> ... <test-suite-N>",
+		Short: "Shows the logs of a related test executions pods.",
+		Long: `Use this command to display the logs of a test executing for given test suite.
+
+To print the status of specific test cases, ` + "`run kyma test logs testSuiteOne testSuiteTwo`" + `.
+Provide at least one test suite name.
+`,
+
+		RunE: func(_ *cobra.Command, args []string) error { return cmd.Run(args) },
+	}
+
+	cobraCmd.Flags().Bool("help", false, "Displays help for the command.")
+	cobraCmd.Flags().StringVar(&o.InStatus, "in-status", defaultLogsInStatus, "Display logs only from testing pods in given status.")
```

Comment:
We need to find a better param name...
maybe "pod-status" or "test-status"?

Reply:
I vote for "test-status" 👍